### ISSUE TITLE
`npm test` fails with `tsc is not recognized`

### ISSUE THREAD

**Issue by Jameskmonger** [alsatian-test/alsatian#545]
```
> alsatian@3.0.0 build:integration-tests C:\Code\alsatian\packages\alsatian
> tsc -p "./test/integration-tests"


lerna ERR! npm run test stderr:
'tsc' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! alsatian@3.0.0 build:integration-tests: `tsc -p "./test/integration-tests"`
npm ERR! Exit status 1
```

This is **immediately** after `npm install && npm run bootstrap`

**Comment by jamesadarich** [comment]
Having a brief look into this may be something to do with the circular dependency. May try bumping tap-barks version of alsatian to trail the current release to see if that helps.

**Comment by jamesadarich** [comment]
resolved in #597
